src/Umbraco.Web.UI.Client/src/packages/user/user-group/workspace/user-group/components/user-group-granular-permission-list.element.ts

```diff
@@ -75,6 +79,7 @@ export class UmbUserGroupGranularPermissionListElement extends UmbLitElement {
 			this._userGroupPermissions.filter((permission) => permission.$type === schemaType) || [];
 
 		(extension.component as any).permissions = permissionsForSchemaType;
+		(extension.component as any).fallbackPermissions = this._userGroupFallbackPermissions;
```

Comment:
For some reason this does not get parsed when I run the code.

As well tried to add a if sentence to prevent the rendering if `_userGroupFallbackPermissions` was not defined, but that never got trickered again.
So I choose to move this to the prop property. Thats is like the right way for us to set props. but that does not have the manifest available, so I only moved this property to that.
Let review that commit. :-)

Reply:
I think it maybe has to do with _userGroupFallbackPermissions can change from the outside, but the permissionsForSchemaType is only changed by this component, so it cannot be changed later. But maybe there is a reactivity problem hidden here.